### PR TITLE
[Edit]Add Codebytes Content/c sharp/concepts/math functions/terms/sqrt/sqrt.md

### PR DESCRIPTION
**Pull request by bimbolaolawale**
### Description

Edited an existing term entry on math.sqrt and math.tanh functions for c-sharp
closes issue number #2982 
### Type of Change

- 
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x ] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by bimbolaolawale** [comment]
@SSwiniarski Please check this out.Thanks

**Comment by SSwiniarski** [comment]
@bimbolaolawale I think you need to look at the source of other entries to see what a codebyte is supposed to look like. These aren't codebytes. They're code grafted on to existing examples. That's not what's being asked for. You need to add something like the following to the end of the entries:

````
## Codebyte Example

Sentence summarizing what the codebyte is doing.

```codebyte/csharp

Code for the codebyte

```
````

Once you correct this, tag me and I'll do a first review.

**Comment by bimbolaolawale** [comment]
@SSwiniarski Thanks for your patience with me and guidance. Please check out the correction and advise on what to do next. Thanks.

File: content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md [hunk 1]
```diff
@@ -75,4 +75,23 @@ public class Example {
     Console.WriteLine("The square root of " + number + " is " + squareRoot);
   }
 }
+
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md:78]:
````suggestion
```

````

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 2]
```diff
@@ -40,5 +40,31 @@ public class Program
...
     // Output: 0.9640275800758169
   }
-}
+
+  ## Codebyte Example
+
+  The codebyte demonstrates the `Math.tanh()` function which returns -1 if the num is negative and if num 
+  is positive it returns 1.
+
+  ```codebyte/csharp
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:49]:
```suggestion
```codebyte/csharp
```

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 3]
```diff
@@ -40,5 +40,31 @@ public class Program
...
+}using System;
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:50]:
```suggestion
using System;
```

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 4]
```diff
@@ -40,5 +40,31 @@ public class Program
...
+        Console.WriteLine("The tanh of num2 = " + tanhvalue);
+
+        tanhvalue = Math.Tanh(num3);
+        Console.WriteLine("The tanh of num3 = " + tanhvalue);
+    }
+  }
+
+
+
+
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:69]:
```suggestion
```

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 5]
```diff
@@ -40,5 +40,31 @@ public class Program
     Console.WriteLine(y);
     // Output: 0.9640275800758169
   }
-}
+
+  ## Codebyte Example
+
+  The codebyte demonstrates the `Math.tanh()` function which returns -1 if the num is negative and if num 
+  is positive it returns 1.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:47]:
```suggestion
  This codebyte demonstrates the `Math.tanh()` function:
```

File: content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md [hunk 6]
```diff
@@ -75,4 +75,23 @@ public class Example {
...
+## Codebyte Example
+
+The code below demonstrates that  when an argument ('x') is negative the method would return Nan with the use of `Math.sqrt()` function.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md:81]:
```suggestion
The code below demonstrates that  when an argument ('x') is negative the `.sqrt()` method would return `NaN`:
```

File: content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md [hunk 7]
```diff
@@ -75,4 +75,23 @@ public class Example {
...
+    public class Example {
+        public static void Main()
+        {
+            double x = -81;
+
+            Console.Write(Math.Sqrt(x));
+        }
+    }
+ }
+
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md:96]:
```suggestion
```codebyte/csharp
using System;

namespace MysquareRoot {
  public class Example {
    public static void Main()
      {
        double x = -81;
        Console.Write(Math.Sqrt(x));
      }
    }
  }
```

File: content/c-sharp/concepts/math-functions/terms/tanh/tanh.md [hunk 8]
```diff
@@ -40,5 +40,31 @@ public class Program
...
+        double tanhvalue = Math.Tanh(num1);
+        Console.WriteLine("The tanh of num1 = " + tanhvalue);
+
+        tanhvalue = Math.Tanh(num2);
+        Console.WriteLine("The tanh of num2 = " + tanhvalue);
+
+        tanhvalue = Math.Tanh(num3);
+        Console.WriteLine("The tanh of num3 = " + tanhvalue);
+    }
+  }
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/tanh/tanh.md:65]:
```suggestion
public class program
{
  public static void Main(string[] args)
  {
    double num1 = 60.0, num2 = 0.0, num3 = 1.0;
    double tanhvalue = Math.Tanh(num1);
    Console.WriteLine("The tanh of num1 = " + tanhvalue);

    tanhvalue = Math.Tanh(num2);
    Console.WriteLine("The tanh of num2 = " + tanhvalue);

    tanhvalue = Math.Tanh(num3);
    Console.WriteLine("The tanh of num3 = " + tanhvalue);
  }
}
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@bimbolaolawale I left a few more comments for you.

**Comment by SSwiniarski** [comment]
@bimbolaolawale Please just edit this branch rather than creating new PRs.

**Comment by bimbolaolawale** [comment]
@SSwiniarski Hello, Please check out the update. Thanks

**Review by SSwiniarski — APPROVED** [review]
@bimbolaolawale I deleted the extra docs folder and now this is ready for a second review.

**Comment by bimbolaolawale** [comment]
Thank you so much for your support 


Sent from Yahoo Mail for iPhone


On Friday, September 1, 2023, 12:18 pm, SSwiniarski ***@***.***> wrote:




@SSwiniarski approved this pull request.

@bimbolaolawale I deleted the extra docs folder and now this is ready for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@bimbolaolawale @SSwiniarski I noticed that the sqrt entry already had a codebyte. The new codebyte didn't really improve on what was there, so I reverted the changes. Looks good to merge

**Pull request merged**